tests/test_encode_decode.py

```diff
@@ -254,9 +256,24 @@ def test_rgb_hsv_round_trip(self, r, g, b):
         self.assertColorsValid(h=(h, h2), s=(s, s2), v=(v, v2))
 
 
+text_strategy = st.text(alphabet=st.characters(blacklist_categories=["Cc", "Cs"]))
+
+json_strategy = st.recursive(
+    st.booleans()
+    | st.integers(min_value=-1 << 63, max_value=1 << 64 - 1)
+    | st.floats(allow_nan=False)
+    | text_strategy,
+    lambda sub_strategy: st.lists(sub_strategy)
+    | st.dictionaries(text_strategy, sub_strategy),
+)
+
+
 class TestPlistlib(unittest.TestCase):
-    # TODO: https://docs.python.org/3/library/plistlib.html
-    pass
+    @no_health_checks
+    @given(payload=json_strategy)
+    def test_dumps_loads(self, payload):
```

Comment:
Let's also have Hypothesis supply the format to dump, and a boolean for whether specify the same format to load or to pass `None` and auto-detect it.